src/commands.rs

```diff
@@ -339,6 +339,22 @@ pub enum OfflineWalletSubCommand {
         /// Selects which policy should be used to satisfy the internal descriptor
         #[structopt(name = "INT_POLICY", long = "internal_policy")]
         internal_policy: Option<String>,
+        /// Optionally create an OP_RETURN output containing given String.
```

Comment:
Nit + I think the limit should apply here too?

```suggestion
        /// Optionally create an OP_RETURN output containing given String in utf8 encoding (max 80 bytes).
```

Reply:
suggestion ack! 
- It is preferable to document that important info, thank you for the suggestion.